pkg/cloudprovider/provider/gce/config.go

```diff
@@ -192,6 +194,12 @@ func newConfig(resolver *providerconfig.ConfigVarResolver, spec v1alpha1.Provide
 		return nil, fmt.Errorf("failed to retrieve gce custom image: %v", err)
 	}
 
+	networkFamily, err := resolver.GetConfigVarStringValue(cpSpec.NetworkFamily)
+	if err != nil {
+		return nil, fmt.Errorf("failed to retrieve network family: %s", err)
+	}
+	cfg.networkFamily = util.NetworkFamily(networkFamily)
```

Comment:
What exactly happens here if network family is an empty string (not set?)

Reply:
Not sure about here. Lemme check.  I just added check for that in the Validation func.

Reply:
Parses just fine. Added a test case.

Reply:
I think we should allow empty string (which would default to IPv4) to ensure backward portability.

Reply:
It will validate and be interpreted as IPv4. In Create we check for dual stack and make adjustments only then so no special handling is needed.